typings/lxml/etree/_cleanup.pyi

```diff
@@ -0,0 +1,21 @@
+# pyright: reportPrivateUsage=false
+
+from __future__ import annotations
+
+from typing import Collection, overload
+
+from .._types import _ElementOrTree, _TagSelector
+
+@overload
+def strip_elements(
```

Comment:
Is `strip_elements` supposed to be here twice?

Reply:
Yes. This is a typing thing (signaled by the `@overload` decorator) where there are two distinct interfaces to a function and you can't represent them both with a single set of annotations. In this case it's because tag names can either be a sequence or you can specify them in variadic form, like:
```python
strip_elements(e, ["nav", "header", "footer"])
# -- OR --
strip_elements( e, "nav", "header", "footer")
```

Reply:
👍 - Sounds good! Just want to double check